pkgs/development/python-modules/wavefile/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, pkgs
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, setuptools
+, pyaudio
+, numpy
+, libsndfile
+}:
+
+buildPythonPackage rec {
+  pname = "wavefile";
+  version = "1.5";
+  format = "setuptools";
+  disabled = pythonOlder "2.0";
+
+  src = fetchFromGitHub {
+    owner = "vokimon";
+    repo = "python-${pname}";
+    rev = "a915ec275bf85602180c1fc95e6919b4d4120d99";
+    sha256 = "9sHj1gb93mCVpejRGSdLJzeFDCeTflZctE7kMWfqFrE=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  buildInputs = [
+    pyaudio
+    numpy
+    libsndfile
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    pyaudio
+    numpy
+    libsndfile
+  ];
+
+  LD_LIBRARY_PATH = lib.makeLibraryPath [ pkgs.libsndfile ];
+
+  patches = [ ./wavefile.py.patch ];
+
+  doCheck = false; # many tests failing
```

Comment:
if you want, you can use `pytestCheckHook` and pass `disabledTestPaths` if a particular file is problematic

Reply:
@jonringer i tried adding

```
  disabledTestPaths = [
    "test/wavefileTest.py"
  ];
```

but it keeps using that file and failing. This is the only test file, which means all test files are failing. So, I will keep doCheck = false, until upstream fixes that i guess